src/sentry/static/sentry/less/shared-components.less

```diff
@@ -2615,7 +2615,7 @@ header + .alert {
     color: @gray-dark;
 
     &:hover,
-    &.active {
```

Comment:
do you think .active was being used anywhere important that this will mess with?